library/src/main/java/com/vanniktech/emoji/EmojiView.java

```diff
@@ -102,6 +111,14 @@ private void handleOnClicks(final ViewPager emojisPager) {
         }
     }
 
+    private void tintIcons() {
+        for (final ImageView emojiTab : emojiTabs) {
+            emojiTab.setColorFilter(themeIconColor, PorterDuff.Mode.SRC_IN);
+        }
+
+        ((ImageView) findViewById(R.id.emojis_backspace)).setColorFilter(themeIconColor, PorterDuff.Mode.SRC_IN);
+    }
+
```

Comment:
It would be better to do this in the layout file, but I don't know of a way of doing so (`android:tint` sadly didn't work). Do you know something?